src/index.tsx

```diff
@@ -1,53 +1,61 @@
 import diff from "microdiff";
 import p5 from "p5";
-import React, { createRef, FC, memo, useState } from "react";
+import React, { createRef, FC, memo, MutableRefObject } from "react";
+import { useRef } from "react";
```

Comment:
```suggestion
import React, { createRef, FC, memo, MutableRefObject, useRef } from "react";
```

Reply:
good suggestion @jhrtn - thanks for that!